src/ResourceManager/DataFactories/Commands.DataFactoryV2/help/Update-AzureRmDataFactoryV2.md

```diff
@@ -0,0 +1,225 @@
+---
+external help file: Microsoft.Azure.Commands.DataFactoryV2.dll-Help.xml
+online version: 
+schema: 2.0.0
+---
+
+# Update-AzureRmDataFactoryV2
+
+## SYNOPSIS
+Updates the properties of a data factory.
+
+## SYNTAX
+
+### ByFactoryName (Default)
+```
+Update-AzureRmDataFactoryV2 [-ResourceGroupName] <String> [-Name] <String> [[-Tag] <Hashtable>]
+ [[-Identity] <FactoryIdentity>] [-Force] [-DefaultProfile <IAzureContextContainer>] [-WhatIf] [-Confirm]
+ [<CommonParameters>]
+```
+
+### ByFactoryObject
+```
+Update-AzureRmDataFactoryV2 [-InputObject] <PSDataFactory> [[-Tag] <Hashtable>] [[-Identity] <FactoryIdentity>]
+ [-Force] [-DefaultProfile <IAzureContextContainer>] [-WhatIf] [-Confirm] [<CommonParameters>]
+```
+
+### ByResourceId
+```
+Update-AzureRmDataFactoryV2 [-ResourceId] <String> [[-Tag] <Hashtable>] [[-Identity] <FactoryIdentity>]
+ [-Force] [-DefaultProfile <IAzureContextContainer>] [-WhatIf] [-Confirm] [<CommonParameters>]
+```
+
+## DESCRIPTION
+The **Update-AzureRmDataFactoryV2** cmdlet updates tags or identity properties of a data factory.
+
+## EXAMPLES
+
+### Example 1
+```
+PS C:\> Update-AzureRmDataFactoryV2 -ResourceGroupName "ADF" -Name "WikiADF" -Tag @{myNewTagName = "myTagValue"}
+
+Confirm
+Are you sure you want to update properties of the data factory 'WikiADF' in resource group 'ADF'?
+[Y] Yes  [N] No  [S] Suspend  [?] Help (default is "Y"): y
+
+
+DataFactoryName   : WikiADF
+DataFactoryId     : /subscriptions/1e42591f-1f0c-4c5a-b7f2-a268f6105ec5/resourceGroups/adf/providers/Microsoft.DataF
+                    actory/factories/wikiadf
+ResourceGroupName : ADF
+Location          : EastUS
+Tags              : {[myNewTagName, myTagValue]}
+Identity          :
+ProvisioningState : Succeeded
+```
+
+This command updates the tags for the factory WikiADF to a dictionary containing a tag named myNewTagName with value myTagValue.
+
+## PARAMETERS
+
+### -DefaultProfile
+The credentials, account, tenant, and subscription used for communication with azure.
+
+```yaml
+Type: IAzureContextContainer
+Parameter Sets: (All)
+Aliases: AzureRmContext, AzureCredential
+
+Required: False
+Position: Named
+Default value: None
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+### -Force
+Don't ask for confirmation.```yaml
```

Comment:
@hvermis make sure to move the trailing ` ```yaml ` string to a line by itself

Reply:
Force is gone